sormas-e2e-tests/src/main/java/org/sormas/e2etests/pages/application/events/EditEventPage.java

```diff
@@ -51,4 +52,17 @@
   public static final By TYPE_OF_PLACE_COMBOBOX = By.cssSelector(" #typeOfPlace div");
   public static final By NEW_TASK_BUTTON = By.id("taskNewTask");
   public static final By EDIT_FIRST_TASK = By.cssSelector("#edit-task-0");
+  public static final By LINK_EVENT_GROUP = By.cssSelector("div#Link\\ event\\ group");
+  public static final By NEW_EVENT_GROUP_RADIOBUTTON =
+      By.xpath("//*[contains(text(),'New event group')]/..");
+  public static final By GROUP_EVENT_NAME_POPUP_INPUT = By.cssSelector(".popupContent #name");
+  public static final By GROUP_EVENT_UUID =
+      By.xpath("//*[contains(text(),'Group id')]/../following-sibling::input[1]");
+  public static final By GROUP_EVENT_UUID2 = By.cssSelector(".popupContent #uuid");
```

Comment:
GROUP_EVENT_UUID  and  GROUP_EVENT_UUID2  are pointing to the same locator. Is there any difference?

Reply:
ye sorry one was not working and tested second one but forgot to delete it :)